src/readBuffer.js

```diff
@@ -1,12 +1,32 @@
+class EarlyEOF extends Error {
+  constructor(message) {
+    super(message);
+    // https://github.com/babel/babel/issues/4485#issuecomment-315569892
+    this.name = EarlyEOF;
```

Comment:
Error.name should be a string, not the class

Reply:
Done